lib/customer.rb

```diff
@@ -0,0 +1,30 @@
+require 'csv'
+require 'pry'
+class Customer
+  attr_reader :id, :email, :address
+  attr_writer :email, :address
```

Comment:
You might as well make `email` and `address` attr_accessor instead of defining this twice.